packages/bindings-test/src/posts/query.rs

```diff
@@ -20,7 +20,7 @@ mod test {
             id: TEST_SUBSPACE_EDITABLE_POST_ID,
             subspace_id: TEST_SUBSPACE,
             section_id: 0,
-            eternal_id: None,
+            external_id: Some("".to_string()),
```

Comment:
Change it since the default of AminoJSON will return empty string "" if the string value is null.